Game.js

```diff
@@ -1,145 +1,454 @@
- let cvs = document.getElementById('canvas');
- let ctx = cvs.getContext('2d')
- const position = document.getElementById('position');
-
- Resize();
- window.addEventListener("resize", Resize);
-
- function Resize() {
-   canvas.width = window.innerWidth;
-   canvas.height = window.innerHeight;
- }
-
- let x = 0, y = 0;
- let sX, sY, k, dx, dy;
- let sXZ, sYZ, K, dxEgg, dyEgg;
-let speedEgg = 4;
-
-
-
- window.onmousemove = function (event) {
-   x = event.pageX;
-   y = event.pageY;
-   //console.log(x,y)
- }
-
- window.onclick = function () {
-   sX = x - Player.x - Player.image.width*0.5*0.5;
-   sY = y - Player.y;
-   k = sX / sY;
-   dy = -Math.sqrt(Math.pow(speedBullet, 2)/(1 + Math.pow(k, 2)));
-   dx = (k * dy);
-   if (Player.activeBullet) {
-     Player.activeBullet--;
-     for (let i = 0; i < bulletRange; i++) {
-       if (Player.bullet[i].isActive) {
-         Player.bullet[i].isActive = false;
-         Player.bullet[i].isDraw = true;
-         Player.bullet[i].x = Player.x + Player.image.width*0.5*0.5;
-         Player.bullet[i].y = Player.y;
-         Player.bullet[i].dx = dx;
-         Player.bullet[i].dy = dy;
-         console.dir(Player.bullet);
-         break;
-       }
-     }
-   }
-   console.log(dx);
-   console.log(dy);
-
- }
+'use strict';
+
+const canvas = document.getElementById('canvas');
+const ctx = canvas.getContext('2d');
+
+window.addEventListener('resize', Resize);
+
+function Resize() {

```

Comment:
С заглавной буквы мы классы и прототипы называем
```suggestion
function resize() {
```